### PR TITLE
[Go] Add gorouter

### PR DESCRIPTION
**Pull request by vardius**
* [x] Please check configuration files
* [x] `neph.yml`, for [neph](http://tbrand.github.io/neph) job processing
* [x] `FRAMEWORKS.yml`, for **frameworks** list
* [x] `travis.yml`, for CI
* [x] `Dockerfile` exists ?
* [x] All tests passes ?

~~~
export FRAMEWORK=gorouter
shards install
shards build --release
bin/neph ${FRAMEWORK}
crystal spec
~~~

### CODE REVIEW

File: FRAMEWORKS.yml [hunk 1]
```diff
@@ -231,6 +231,10 @@ go:
     website: www.gorillatoolkit.org/pkg/mux
     version: "1.7"
     language: "1.12"
+  gorouter:
+    website: github.com/vardius/gorouter/wiki
```
Comment (waghanza) [inline FRAMEWORKS.yml]: perhaps better `website: rafallorenz.com/gorouter`

File: go/gorouter/main.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    params, _ := gorouter.FromContext(r.Context())
+    id := params.Value("id")
+    w.Write([]byte(id))
+}
+
+func main() {
+    router := gorouter.New()
+
+    router.GET("/", http.HandlerFunc(index))
+    router.GET("/user/{id}", http.HandlerFunc(userId))
```
Comment (waghanza) [inline go/gorouter/main.go]: can this route be constrained by allowing only digits ?

File: go/gorouter/go.mod [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module main
+
+require (
```
Comment (waghanza) [inline go/gorouter/go.mod]: is it not a best-practice in `go` to specify the version in `go.mod` ?

File: go/gorouter/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+FROM golang:1.12.5
+
+WORKDIR /go/src/app
+COPY main.go go.mod ./
+
+ENV GO111MODULE=on
```
Comment (waghanza) [inline go/gorouter/Dockerfile]: is it required in **go** `1.12` ?

File: FRAMEWORKS.yml (hunk 1 above)
Comment (vardius) [reply FRAMEWORKS.yml]: I wanted to link to router directly, wiki is always up to date

File: go/gorouter/Dockerfile (hunk 4 above)
Comment (vardius) [reply go/gorouter/Dockerfile]: I copied docker image from other go routers in this repo

Comment (waghanza) [reply go/gorouter/Dockerfile]: yeah, but some are outdated

File: go/gorouter/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,12 @@
+FROM golang:1.12.5
+
+WORKDIR /go/src/app
+COPY main.go go.mod ./
```
Comment (appleboy) [inline go/gorouter/Dockerfile]: missing `go.sum` file?

Comment (vardius) [reply go/gorouter/Dockerfile]: added go.sum

Comment (waghanza) [reply go/gorouter/Dockerfile]: @appleboy Is `go.sum` really required for a benchmark use-case ?

**Pull request merged**